### ISSUE TITLE
Xeroscaping Photosharing site

### ISSUE THREAD

**Issue by mapineda** [atxhack4change/2016-project-proposals#11]
## Short description of the idea (140 character tweet):
Xeroscaping education and awareness application disguised as a photosharing mobile app

_Your short answer_

## What is your project concept or idea? What challenge or opportunity will it solve?
Local residents are unaware of the tax breaks, subsidies and incentives the city gives to convert their garden/lawn to xeroscaping. This in turn conserves millions of gallons of water.

_Your answer_

## Who will benefit from your project? Describe the humans at the center of the problem - who are they?
- Local Travis County Residents 
### How would they benefit? Can you tell their story?
- Users will benefit by sharing their home landscaping projects with their community. There is also a financial benefit for the users since the city provides economic incentive to convert their landscapes to xeroscaping.
### Do you have photos of them? Or quotes?
- not yet.

_Your answer_

### [Look at the right sidebar :arrow_upper_right: ! Do you see the Labels? Pick a few]

- Arts & Culture
- Economic Development
- Economic Opportunity
- Education
- Environment
- Sustainability

## What is the current state of your project idea? [Pick more labels :arrow_upper_right: ]

- I have an idea or **concept** of a solution, but have not yet prototyped or tested it

## Tell us more about the current state of your project idea.
Ideation stage
### Do you have any links to text, mockups, code, or data?

_Your answer_

## Tell us about yourself. 
Software Developer passionate about the environment and innovative tech.
### What is your background? Why is this challenge important to you?
I am a fullstack software developer. I am passionate about the environment and that includes water conservation.
### Are you representing a group in submitting this project idea?
Representing the group from Code Across 2016 Xeroscaping

_Your answer_

**Comment by Rogue-Aider** [comment]
Yeah! I've been thinking about this. A "Xeriscape Crawl", checking out some of the more inventive and good looking xeriscapes.

**Comment by mateoclarke** [comment]
Thanks for adding this project idea @mapineda. I've added some labels based on your answers. I thought it might also be helpful to reference the materials this team gathered from Code Across. You might include this as a link: https://github.com/open-austin/project-ideas/issues/59

**Comment by mateoclarke** [comment]
You might also want to include a link to a recent article that gives some context about local watering restrictions. 
http://www.austinmonitor.com/stories/2016/05/council-approves-water-restrictions/
Here is a quote from Mayor Adler that supports the spirit of this project.
> Mayor Steve Adler weighed in near the end of the debate, saying that other cities throughout the southwest were moving toward embracing native landscapes that require less water than turf lawns. He pointed out that the ordinance will allow for unlimited use of alternative methods of irrigation, including hose-end or drip-end watering.

**Comment by ssharif1** [comment]
### ACTION NEEDED 

Hello Project Champion, 

Please provide the following:
Full Name + Email Address 
You can comment here or send to ssharif1@stedwards.edu 

We are going to be sending out a note + newsletter to our 345 (as of May 18) attendees sharing our submitted projects before the event and your hackers will want to know who you are. Additionally, we have PC-specific communication that we want to make sure that you are included on.

Thanks,
ATX H4C Staff

**Comment by mateoclarke** [comment]
Hi @mapineda! You're project has been accepted!

Now that your project has been approved, here are some things we suggest you do next:
https://github.com/atxhack4change/2016-project-proposals#once-your-project-is-accepted-here-are-some-things-we-suggest-you-do-next

You will receive an email in the next few days asking you to confirming your speaking slot for the Saturday morning pitches. 

Please let us know if you will no longer be able to champion your project at the hackathon.

**Comment by danielhonker** [comment]
@mapineda - you should have received an invite to join Slack. If you haven't already joined the team and seen the Slack channel for your project, it's #xeriscape. We'll be sending out an email with details on pitches soon.

**Comment by mapineda** [comment]
@danielhonker cool, thanks. I've joined the slack!

**Comment by mapineda** [comment]
Thinking about coding this up in Ionic/Cordova. Would like to begin the design process. Lean UX Design. Low-res wireframing in the next few days.
